TravelBug/src/Components/Sidebar/Sidebar.jsx

```diff
@@ -10,7 +10,9 @@ const Sidebar = () => {
           <div className="component-child">
             <i class="fas fa-home"></i>
```

Comment:
- you can change class of <i></> too in className. other wise it will give error in console

Reply:
Done